docs/content/plugins/discoverability.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Plugin Discoverability
+description: Learn more about searching and broadcasting available plugins for Porter
+---
+
+## Search
+
+Porter maintains a list of plugins that users can search via
+`porter plugin search [NAME]`. If no name is supplied, the full listing will be
+returned.  See the help menu for all command options: `porter plugin search -h`.
+
+For example, here we search for an `azure` plugin:
+
+```console
+$ porter plugin search azure
+Name    Description                                    Author           URL                                      URL Type
+azure   A plugin for utilizing Azure cloud resources   Porter Authors   https://cdn.porter.sh/plugins/atom.xml   Atom Feed
+```
+
+## Broadcast
+
+To add your plugin to the list, add an entry to the plugin directory
+[list](https://github.com/deislabs/porter/blob/master/pkg/plugins/directory/index.json)
+with all the pertinent informational fields filled out.
+
+For instance, a new entry might look like:
+
+```json
+  {
+    "name": "myplugin",
+    "author": "My Name",
+    "description": "A plugin for doing great things",
+    "URL": "https://github.com/org/project/releases/download",
+  },
+```
+
+The `URL` field should either be an Atom Feed URL (for example, Porter uses
```

Comment:
This is a bit hard to read with the parentheticals. I think people could take this in easier with some bullet points. ALL THE BULLET POINTS

The `URL` field should be one of the following:

* **Atom Feed URL:** Porter uses the following...
* **Download URL:** Directory where binaries are hosted, such as GitHub releases `https://github.com/org/project/releases/download`

Reply:
* Good call
* Definitely reads better
* Now featuring bullet points!

Reply:
I took a technical writing class in college to meet a writing requirement to graduate. While everyone else wrote 20pg papers on great english literature, I learned that people don't read, use moar bullet points. 😀